src/cli.rs

```diff
@@ -589,6 +599,27 @@ pub fn run(args: Opt) -> Result<(), Box<dyn Error>> {
                         force_update,
                     )
                 }
+                VersionCommand::BumpBreaking {
+                    pkg_opts,
+                    force_update,
+                } => {
+                    let predicate = make_pkg_predicate(pkg_opts)?;
+                    commands::set_version(
+                        &ws,
+                        |p| predicate(p),
+                        |p| {
+                            let mut v = p.version().clone();
+                            v.pre = Vec::new();
+                            if v.major == 0 {
```

Comment:
There's also a Cargo/Rust-specific extension to SemVer where the versions `0.0.X` are treated as public API incompatible, so it might make sense to have that in mind and possibly use that here as well, since we're talking about Cargo packages anyway.

Reply:
is that documented somewhere? I don't know about any specialty in semver itself regarding that.

Reply:
https://doc.rust-lang.org/cargo/reference/semver.html#change-categories

> This guide uses the terms "major" and "minor" assuming this relates to a "1.0.0" release or later. Initial development releases starting with "0.y.z" can treat changes in "y" as a major release, and "z" as a minor release. "0.0.z" releases are always major changes. This is because Cargo uses the convention that only changes in the left-most non-zero component are considered incompatible.